webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -51,20 +51,23 @@ export class TasksComponent implements OnInit, OnDestroy {
   private debouncedReload;
 
   private subscribeToStore() {
-    const reduxSubscription = combineLatest(
+    const assignment$ = this.store
+      .select(Selectors.getTasksLoaded)
+      .subscribe(tasksLoaded => this.tasksLoaded = tasksLoaded);
+    this.subscription.add(assignment$);
+
+    const taskList$ = combineLatest(
       this.store.select(Selectors.getTasksList),
-      this.store.select(Selectors.getTasksLoaded),
       this.store.select(Selectors.getSelectedTask),
     ).subscribe(([
```

Comment:
I cannot make a code suggestion because of the deleted line but this should be:
```ts
const taskList$ = combineLatest([
  this.store.select(Selectors.getTasksList),
  this.store.select(Selectors.getSelectedTask),
]).subscribe(([
```
notice the `[]` brackets, the brackets-less signature [has been deprecated](https://rxjs.dev/deprecations/array-argument) and will be removed in v8